modules/dag-l1/src/main/scala/org/tessellation/dag/l1/kryo/package.scala

```diff
@@ -15,10 +12,7 @@ package object kryo {
     classOf[Proposal] -> 1002,
     classOf[BlockProposal] -> 1003,
     classOf[CancelledBlockCreationRound] -> 1004,
-    classOf[UUID] -> 1005,
```

Comment:
was UUID registered twice?

Reply:
Yes, once in `shared` and secondly here. Also, I think we should have a rule, that we register all classes that aren't part of the current module only in the `shared` module. That would include all JDK and Scala and library classes.